src/billing/components/PayAsYouGo/CancelServiceReasonsForm.tsx

```diff
@@ -123,15 +123,16 @@ function CancelServiceReasonsForm() {
               checked={canContactForFeedback}
               onChange={toggleCanContactForFeedback}
               size={ComponentSize.Small}
-              titleText="I understand and agree to these conditions"
+              titleText="I am open to talking further with the Influx product &amp; design"
```

Comment:
I'd double check this since I'm not sure how clockface handles HTML entities being passed in as strings

```suggestion
              titleText="I am open to talking further with the Influx product & design"
```

Reply:
`&` isn't working but `&amp;` is what worked for `InputLabel`